Sources/RealDeviceMapLib/Misc/MemoryCache.swift

```diff
@@ -27,7 +27,11 @@ public class MemoryCache<T> {
                     let hit = self.hits[element.key]
                     return hit != nil && now.timeIntervalSince(hit!) < keepTime
                 }
+                let keys = self.store.keys
                 self.lock.unlock()
+                self.hits = self.hits.filter { element in
```

Comment:
@123FLO321 what do you think about that change? Some people reported a memory leak to me. especially this hits list would increase in time significantly with pokemon and incident cache

Reply:
Looks good to me.